src/live_restore/live_restore_server.c

```diff
@@ -193,6 +199,17 @@ __live_restore_worker_run(WT_SESSION_IMPL *session, WT_THREAD *ctx)
     WTI_LIVE_RESTORE_SERVER *server = S2C(session)->live_restore_server;
     uint64_t time_diff_ms;
 
+    if (!F_ISSET(S2C(session), WT_CONN_READY)) {
```

Comment:
If we're dependent on this flag is there an option to start the server once the flag is set?

Reply:
We could, but it would require moving`__wt_live_restore_server_create` out of `__wti_connection_workers` where we start everything else and placing it right at the end of the `conn_open` logic. That feels messy to me